pom.xml

```diff
@@ -464,6 +466,8 @@
     <version.org.jacoco>0.6.3.201306030806</version.org.jacoco>
 -->
     <version.org.sonatype.plugins.nexus-staging-maven-plugin>1.6.6</version.org.sonatype.plugins.nexus-staging-maven-plugin>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Its fine to leave it here, it would help to show what it was added for if we need to look in future.